skills/csharp/experimental/itsmskill/Extensions/ActivityEx.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Extensions
+{
+    using System;
+    using ITSMSkill.Extensions.Teams.TaskModule;
+    using Microsoft.Bot.Schema;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// ActivityEx for purpose of TeamsTaskModule
+    /// </summary>
+    public static class ActivityEx
+    {
+        public const string FetchType = "task/fetch";
```

Comment:
like we talked about, please create an issue in SDK so that they define enum for these protocol strings so we're always aligned with the protocol used by SDK.

before that, can these strings be private since they're not being used directly from outside this class?

Reply:
same for all these const strings

Reply:
Created issue for this and added a TODO comment to remove ActivityEx when the support is added to SDK: https://github.com/microsoft/botframework-sdk/issues/5816